biggus/_init.py

```diff
@@ -66,7 +30,13 @@
 import numpy.ma as ma
 
 
-__version__ = '0.13.0'
+__all__ = []
+
+
+def export(defn):
+    globals()[defn.__name__] = defn
+    __all__.append(defn.__name__)
+    return defn
```

Comment:
Thanks to @dabeaz for this [neat little trick](http://www.dabeaz.com/modulepackage/ModulePackage.pdf).